includes/Modules/Analytics.php

```diff
@@ -1102,15 +1054,6 @@ function( $property ) use ( $urls ) {
 					}
 
 					$result = array_merge( $response, $properties );
-
-					if ( $existing_tag ) {
-						$result = array_merge( $result, array( 'existingTag' => $has_access_to_property ) );
-					}
-
-					if ( $matched_property ) {
-						$result = array_merge( $result, array( 'matchedProperty' => $matched_property ) );
```

Comment:
All other occurrences of the `$matched_property` variable should be removed, since it's no longer used anymore now.

Reply:
removed https://github.com/google/site-kit-wp/pull/311/commits/c7de346fd25e666f7e01929fbf0b7884276fb1d0